packages/@o3r/core/schematics/ng-add/index.ts

```diff
@@ -1,37 +1,12 @@
 import { chain, externalSchematic, Rule, SchematicContext, Tree } from '@angular-devkit/schematics';
-import { NodePackageInstallTask } from '@angular-devkit/schematics/tasks';
-import { NodePackageName } from '@angular-devkit/schematics/tasks/package-manager/options';
 import * as fs from 'node:fs';
 import * as path from 'node:path';
 import { lastValueFrom } from 'rxjs';
 import type { PackageJson } from 'type-fest';
 import { NgAddSchematicsSchema } from './schema';
 import { displayModuleList } from '../rule-factories/module-list';
-
-/**
- * Install dev dependency on your application
- *
- * Note: it should not be moved to other packages as it should run before the installation
- * of peer dependencies
- */
-class DevInstall extends NodePackageInstallTask {
-  public quiet = false;
-
-  /** @inheritdoc */
-  public toConfiguration() {
-    const installOptions = process.env?.npm_execpath?.includes('yarn')  ? 'yarn' : 'npm';
-    return {
-      name: NodePackageName,
-      options: {
-        command: 'install',
-        quiet: this.quiet,
-        workingDirectory: this.workingDirectory,
-        packageName: `${this.packageName!} ${installOptions === 'yarn' ? '--prefer-dev' : '-D'}`,
-        packageManager: installOptions
-      }
-    };
-  }
-}
+import { presets } from './presets';
+import { DevInstall } from './presets/helpers';
```

Comment:
I find it a bit weird that it would be in the folder presets. I don't mind it being moved but I don't see why that would belong there :)